core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -153,6 +156,14 @@ class BlockManagerMasterEndpoint(
       removeExecutor(execId)
       context.reply(true)
 
+    case DecommissionBlockManagers(executorIds) =>
+      val bmIds = executorIds.flatMap(blockManagerIdByExecutor.get)
+      decommissionBlockManagers(bmIds)
```

Comment:
It seems that we don't need to declare `bmIds`.
```
-      val bmIds = executorIds.flatMap(blockManagerIdByExecutor.get)                                               │
-      decommissionBlockManagers(bmIds)                                                                            │
+      decommissionBlockManagers(executorIds.flatMap(blockManagerIdByExecutor.get))
```

Reply:
refactored.